index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
```

Comment:
Please wrap your header items in a header tag.

Reply:
I suggest you give an explanation on why you requested the changes above. it might be obvious for you but not for someone else.
Please refer to the lesson[How to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)
<img width="1654" alt="Screenshot 2022-10-31 at 12 05 31" src="https://user-images.githubusercontent.com/48546992/198986912-a045b546-b335-4a37-9e8d-67aac44bc08d.png">